std/traits.d

```diff
@@ -8391,6 +8391,16 @@ template getSymbolsByUDA(alias symbol, alias attribute)
     static assert(getSymbolsByUDA!(A, Attr).stringof == "tuple(a, a, c)");
 }
 
+// #18884: getSymbolsByUDA fails on AliasSeq members
+@safe unittest
+{
+    struct X {
+        alias A = AliasSeq!(ulong, uint);
+    }
+
+    assert(__traits(compiles, getSymbolsByUDA!(X, X)));
```

Comment:
could be a `static assert`. More important insert of checking that it compiles you could check the expected results.